edu-services/cm-service/cm-impl/hibernate-impl/impl/src/java/org/sakaiproject/coursemanagement/impl/CourseManagementServiceSampleChainImpl.java

```diff
@@ -191,6 +191,10 @@ public Map findSectionRoles(String userEid) {
 		return null;
 	}
 
+	public Map findSectionRolesMatchingRoles(String userEid, List<String> roles) {
```

Comment:
missing generics and instead return null for a collection consider returning an empty unmodifiable map.